kythe/typescript/indexer.ts

```diff
@@ -1241,14 +1241,25 @@ class Visitor {
           console.error(`TODO: export ${exp.name} has no symbol`);
           continue;
         }
-        // TODO: import a type, not just a value.
         const remoteSym = this.typeChecker.getAliasedSymbol(localSym);
+        const anchor = this.newAnchor(exp.name);
         const kExport = this.host.getSymbolName(remoteSym, TSNamespace.VALUE);
```

Comment:
It's not guaranteed that the export has a value.  For example, interface and type alias.

You can check:
`(flags & ts.SymbolFlags.Value) !== 0`

Be careful to check on the actual symbol, and not the alias of it.